cssselect/xpath.py

```diff
@@ -272,6 +272,12 @@ def xpath_negation(self, negation):
         else:
             return xpath.add_condition('0')
 
+    def xpath_relation(self, relation):
+        xpath = self.xpath(relation.selector)
+        combinator, subselector, *_ = relation.subselector
```

Comment:
This fails on Python 2, though I think we are fine with dropping Python 2 from this project.